.github/workflows/e2e-kubernetes.yaml

```diff
@@ -8,28 +8,28 @@ on:
 
 jobs:
   end-to-end:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
         TEST_GROUP: [smoke, es, cassandra, streaming, examples1, examples2, generate, upgrade, istio]
     steps:
     - uses: actions/setup-go@v2.1.3
       with:
-        go-version: '1.14.4'
-    - uses: jpkrohling/setup-minikube@v1.0.2
+        go-version: '1.14.12'
+    - uses: actions/checkout@v2.3.4
+    - name: "setup docker"
+      run: ./.ci/setup-docker.sh
+    - uses: manusa/actions-setup-minikube@v2.4.1
```

Comment:
nice I didn't see this. In the future we could also consider using `kind`.

Reply:
Indeed, the new e2e tests are using kind and kuttl :-)